### PR TITLE
Add CentoOS Origin repo for 310 release

### PR DESCRIPTION
**Pull request by DanyC97**
@vrutkovs pls help review

### CODE REVIEW

**Comment by DanyC97** [comment]
@sdodson @michaelgugino fyi, thx

File: roles/openshift_repos/tasks/centos_repos.yml [hunk 1]
```diff
@@ -19,6 +19,6 @@
     src: "{{ item }}"
     dest: "/etc/yum.repos.d/{{ (item | basename | splitext)[0] }}"
   with_first_found:
-    - "CentOS-OpenShift-Origin{{ ((openshift_version | default('')).split('.') | join(''))[0:2] }}.repo.j2"
+    - "CentOS-OpenShift-Origin{{ ((openshift_release | default('')).split('.') | join(''))[0:3] }}.repo.j2"
```
Comment (michaelgugino) [inline roles/openshift_repos/tasks/centos_repos.yml]: openshift_release is not guaranteed to be defined.  Need to continue using openshift_version here.

Comment (DanyC97) [reply roles/openshift_repos/tasks/centos_repos.yml]: @michaelgugino can you pls sync up with @vrutkovs and then i'll happily change? And will be good before bot will merge https://github.com/openshift/openshift-ansible/pull/9548

Comment (vrutkovs) [reply roles/openshift_repos/tasks/centos_repos.yml]: Yeah, I probably was wrong - lets revert to `openshift_version`, sorry about that

Comment (DanyC97) [reply roles/openshift_repos/tasks/centos_repos.yml]: okay @vrutkovs so i guess we should have

`((openshift_version | default('')).split('.') | join(''))[0:3] }}` ? note `.3` vs `.2`

**Comment by DanyC97** [comment]
updated @vrutkovs @michaelgugino , thank you

File: roles/openshift_repos/tasks/centos_repos.yml [hunk 2]
```diff
@@ -19,6 +19,6 @@
     src: "{{ item }}"
     dest: "/etc/yum.repos.d/{{ (item | basename | splitext)[0] }}"
   with_first_found:
-    - "CentOS-OpenShift-Origin{{ ((openshift_version | default('')).split('.') | join(''))[0:2] }}.repo.j2"
+    - "CentOS-OpenShift-Origin{{ ((openshift_version | default('')).split('.') | join(''))[0:3] }}.repo.j2"
```
Comment (michaelgugino) [inline roles/openshift_repos/tasks/centos_repos.yml]: This change is incorrect.
No need to change this at all.

**Comment by DanyC97** [comment]
@michaelgugino updated, thanks

**Pull request merged**